v2/dtos/command.go

```diff
@@ -0,0 +1,27 @@
+//
+// Copyright (C) 2020 IOTech Ltd
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package dtos
+
+import "github.com/edgexfoundry/go-mod-core-contracts/v2/models"
+
+// Command defines a specific read/write operation targeting a device
+// This object and its properties correspond to the Command object in the APIv2 specification:
+// https://app.swaggerhub.com/apis-docs/EdgeXFoundry1/core-metadata/2.x#/Command
+type Command struct {
+	Id   string `json:"id" yaml:"id,omitempty"`     // Id is a unique identifier, such as a UUID
+	Name string `json:"name" yaml:"name,omitempty"` // Command name (unique on the profile)
```

Comment:
Name should be required since it is used in a Add request, also need to validate that get and put are not both false.

Reply:
Added tags:
* name:  validate:"required"
* Get:  validate:"required_without=Put"
* Put:  validate:"required_without=Get"